recipes/lml/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "lml" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "12783c8851bb1b1aacf6d139aa41fb529e1723bff863e71034af07a55bf1e5a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - lml
+  requires:
+    - nose
+
+about:
+  home: https://github.com/chfw/lml.git
+  license: New BSD
```

Comment:
Change to `BSD-3-Clause` as per:

https://opensource.org/licenses/alphabetical